devtools/project-core-extension-codestarts/src/test/java/io/quarkus/devtools/codestarts/quarkus/FunqyHttpCodestartTest.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.devtools.codestarts.quarkus;
+
+import static io.quarkus.devtools.codestarts.quarkus.QuarkusCodestartCatalog.Language.JAVA;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.devtools.testing.codestarts.QuarkusCodestartTest;
+
+public class FunqyHttpCodestartTest {
+
+    @RegisterExtension
+    public static QuarkusCodestartTest codestartTest = QuarkusCodestartTest.builder()
+            .codestarts("funqy-http")
+            .languages(JAVA)
+            .build();
+
+    @Test
+    void testContent() throws Throwable {
+        codestartTest.checkGeneratedSource("org.acme.MyFunctions");
+        codestartTest.checkGeneratedTestSource("org.acme.MyFunctionsTest");
+        codestartTest.checkGeneratedTestSource("org.acme.MyFunctionsIT");
+        //codestartTest.buildAllProjects();
```

Comment:
This line is commented in multiple classes, should it be remove ?

Reply:
Yeah, I use it to make sure the codestart is building, but it's then done in the IT by CI..